Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Culture innovations/history

## How to test:
- Run in-game and tell us your opinion.
- Try and keep an eye out for when the date hits 605.1.1 to see if the AI gets their new techs properly. 
- We have dates for techs before 583, but for now, they are necessary to look at for obvious reasons.

## Changelog:

- Added new history for every race.
     - New dates for when to unlock certain techs.
     - Gave the appropriate amount of techs to cultures based on their tech level in lore.
     - Each culture has a unique history where they will unlock certain techs throughout the new dates.
